src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -170,6 +170,13 @@ export class RundownComponent implements OnInit, OnDestroy, OnChanges {
     return action ? { ...actionMap, [segment.id]: action } : actionMap
   }
 
+  public get isActiveOrRehearsal(): boolean {
+    if (!this.rundown) {
+      return false
+    }
+    return (this.rundown && this.rundown.mode === RundownMode.ACTIVE) || this.rundown.mode === RundownMode.REHEARSAL
```

Comment:
`this.rundown &&` can be removed. The truthiness of the rundown value has already been checked in the guard on line 174-176.

To condense the condition a bit more, you can use `Array.includes`:
```typescript
    return [RundownMode.ACTIVE, RundownMode.REHEARSAL].includes(this.rundown.mode)
```